framework/helpers/BaseHtml.php

```diff
@@ -1362,6 +1364,21 @@ public static function activeTextInput($model, $attribute, $options = [])
         return static::activeInput('text', $model, $attribute, $options);
     }
 
+    /**
+     * Generate placeholder from model label.
+     * @param Model $model the model object
+     * @param string $attribute the attribute name or expression. See [[getAttributeName()]] for the format
+     * about attribute expression.
+     * @param array $options the tag options in terms of name-value pairs. These will be rendered as
+     * the attributes of the resulting tag. The values will be HTML-encoded using [[encode()]].
```

Comment:
Should this not work the same as the standard activeLabel method and call getAttributeName before calculating the label so that it copes with forms with multiple models?

attributes like [0]name, [1]name will not get the correct label.

Reply:
Could you create a separate issue?